package.json

```diff
@@ -28,13 +28,13 @@
     "watch:jade": "chokidar 'src/index.jade' -c 'npm run build:jade'",
     "watch:scss": "chokidar 'src/scss/**/*.scss' -c 'npm run build:scss'",
     "test": "npm run test:sass",
-    "test:sass": "sass-lint -c node_modules/mofo-style/.sass-lint.yml --verbose --no-exit src/scss/**/*.scss",
+    "test:sass": "sass-lint -c .sass-lint.yml --verbose --no-exit src/scss/**/*.scss",
```

Comment:
There isn't a `.sass-lint.yml` in this project, so I think this is just falling back to defaults? If that's what we want then you can remove the `-c .sass-lint.yml` part.